### ISSUE TITLE
Using fitnesse-fixtures:3.26.0, how to configure Firefox to use a proxy

### ISSUE THREAD

**Issue by violinner** [fhoeben/hsac-fitnesse-fixtures#225]
Assuming a proxy is running on localhost on port 8188, 
is it possible to configure Firefox version 60 to use a proxy?
---
|start driver for | firefox | with profile| !{ some settings } |

I need to set
- Manual proxy configuration = 1
- HTTP Proxy =  localhost
- HTTP Proxy port = 8188
---

Using Firefox version < 50 and older HSAC Fixtures, this was possible using DesiredCapabilities.

**Comment by fhoeben** [comment]
I've never set a proxy this way, but it might be an issue that this way all properties are passed as strings.

I would recommend try creating the profile using MapFixture, which allows you to set int values, but I just discovered that 'selenium driver setup' (or actually `LocalDriverFactory`) always sets all profile values as string. If the problem is indeed that some values should be integers or booleans we could change the code to pass the values based on the type of value in the map.

If such a change were made you could do something like:
```
|script|map fixture|
|set int value |2|for|network.proxy.type|
|set value |localhost|for|network.proxy.http|
|set int value|8188|for|network.proxy.http_port|
|$profile=|copy map|

|script |selenium driver setup|
|start driver for|firefox | with profile |$profile|
```

**Comment by fhoeben** [comment]
I pushed some changes (to the master branch) that should allow you to set integer or boolean preference values in a firefox profile...

**Comment by violinner** [comment]
---
The example produces a non-functional line in prefs.js:
user_pref("network", "{proxy={type=2, http=localhost, http_port=8188}}");

This is needed:
user_pref("network.proxy.type=2)
user_pref("network.proxy.http=localhost)
user_pref("network.proxy.http_port=8188)
---

**Comment by fhoeben** [comment]
You are right, I'm sorry. We should have disabled the period processing of map fixture.

Try it again with adding an extra line to script table with map fixture:
```
|script|map fixture|
|expand periods in names to nested maps|false|
|set int value |2|for|network.proxy.type|
|set value |localhost|for|network.proxy.http|
|set int value|8188|for|network.proxy.http_port|
|$profile=|copy map|

|script |selenium driver setup|
|start driver for|firefox | with profile |$profile|
```
